draft-ietf-quic-recovery.md

```diff
@@ -446,14 +433,15 @@ flight, since this packet adds network load without establishing packet loss.
 ## Generating Acknowledgements
 
 QUIC SHOULD delay sending acknowledgements in response to packets, but MUST NOT
-excessively delay acknowledgements of packets containing frames other than ACK
-or ACN_ECN.  Specifically, implementations MUST attempt to enforce a maximum
-ack delay to avoid causing the peer spurious timeouts.  The RECOMMENDED maximum
-ack delay in QUIC is 25ms.
-
-An acknowledgement MAY be sent for every second full-sized packet, as TCP does
-{{?RFC5681}}, or may be sent less frequently, as long as the delay does not
-exceed the maximum ack delay.  QUIC recovery algorithms do not assume the peer
+excessively delay acknowledgements of packets containing frames other than ACK.
+Specifically, implementations MUST attempt to enforce a maximum
+ack delay to avoid causing the peer spurious timeouts.  The maximum ack delay
+is communicated in the `max_ack_delay` transport parameter and the default
+value is 25ms.
+
+An acknowledgement SHOULD be sent immediately upon receiving two or more
+retransmitable full-sized packets, as TCP does {{?RFC5681}}.  QUIC recovery
+algorithms, including tail loss probe{{tail-loss-probe}}, assume the peer
```

Comment:
How do you define full-sized packets when this depends on PMTU discovery?
The text also isn't very clear about receiving drips of full sized packets: can you ack after only one full packet if it exceeds max ACK delay? I'd say obviously yes, but I'm sure the text is very clear (before or ofter this change).